app/controllers/users_controller.rb

```diff
@@ -43,6 +43,20 @@ def enroll
     end
   end
 
+  ####################################################
+  # Page for showing course info for particular user #
+  ####################################################
+  def show
+      @user = User.find_by_username(params[:username])
```

Comment:
This should conform to the project's Rubocop style rules. Use two spaces for indentation.